### PR TITLE
Release 5 v2 apps

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

The PR https://github.com/SumoLogic/sumologic-documentation/pull/3239, and https://github.com/SumoLogic/sumologic-documentation/pull/3240 are merged and created this new single PR.

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions, .clabot
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/integrations/webhooks/hasura.md [hunk 1]
```diff
@@ -7,6 +7,9 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 <img src={useBaseUrl('img/integrations/webhooks/hasura-logo.png')} alt="Thumbnail icon" width="50"/>
 
+
+The Hasura app for Sumo Logic enables you to seamlessly integrate and analyze your Hasura data event and schema operation metrics. It offers a holistic view of your data events, showcasing key metrics such as Events Generated, distribution by Operation, Deleted events, Target Schema, and Target Tables. By tracking trends in Events by Table Schema, Trigger, and Operation, you can make data-driven decisions to optimize your data architecture and operations. Additionally, the app provides a quick overview of recent data changes with summaries for Recent Insertions, Updates, and Deletions, allowing for real-time monitoring and informed action.
```
Comment (kimsauce) [inline docs/integrations/webhooks/hasura.md:11]:
```suggestion
The Hasura app for Sumo Logic enables you to seamlessly integrate and analyze your Hasura data event and schema operation metrics. It offers a holistic view of your data events, showcasing key metrics such as Events Generated, Distribution by Operation, Deleted events, Target Schema, and Target Tables. By tracking trends in Events by Table Schema, Trigger, and Operation, you can make data-driven decisions to optimize your data architecture and operations. Additionally, the app provides a quick overview of recent data changes with summaries for Recent Insertions, Updates, and Deletions, allowing for real-time monitoring and informed action.
```

File: docs/integrations/webhooks/ilert.md [hunk 2]
```diff
@@ -63,3 +113,17 @@ Follow the steps to configure the iLert Webhook.
 - For detailed information about webhook creation, refer to the [iLert Documentation](https://docs.ilert.com/integrations/webhook).
 - For support, [contact iLert](https://docs.ilert.com/contact).
 :::
+
+### Installing the iLert app
+
+{@import ../../reuse/apps/app-install-v2.md}
+
+## Viewing iLert dashboard
```
Comment (kimsauce) [inline docs/integrations/webhooks/ilert.md:121]: I'd keep this plural (even though there's only one dashboard) for consistency with the rest of the corresponding app docs. Kind of like these headers, where they're plural no matter what.
Screenshot 2023-12-22 at 5 16 40 AM


```suggestion
## Viewing iLert dashboards
```

File: docs/integrations/webhooks/mailgun.md [hunk 3]
```diff
@@ -7,12 +7,73 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 <img src={useBaseUrl('img/integrations/webhooks/mailgun-logo.png')} alt="Thumbnail icon" width="50"/>
 
+
+The Mailgun app for Sumo Logic enables you to seamlessly monitor email system performance and user interactions through two comprehensive dashboards. The provided dashboard empowers you to analyze and optimize email delivery by monitoring event trends, identifying errors, understanding client behaviour, and tracking event geolocation data, ensuring efficient email management and an enhanced user experience. Meanwhile, the dashboard also offers a holistic view of email delivery, allowing you to track issues, complaint patterns, and geographic trends in events and complaints, facilitating effective email management and optimization.
```
Comment (kimsauce) [inline docs/integrations/webhooks/mailgun.md:11]: The first sentence says there are two dashboards, but only one is referenced in this paragraph

Comment (JV0812) [reply docs/integrations/webhooks/mailgun.md:11]: This app has two dashboard one is the overview (info added this in the second line of paragraph) and the other is Failed, Complained Events which is covered in the last line of the para.

File: docs/integrations/webhooks/mailgun.md [hunk 4]
```diff
@@ -47,3 +108,23 @@ Follow the below steps to configure the Mailgun webhook.
...
+
+{@import ../../reuse/apps/view-dashboards.md}
+
+### Overview
+
+The **Mailgun - Overview** dashboard offers a comprehensive overview of email system performance and user interactions, featuring panels that include Log Level, Total Events, Total Error Events, Log Level Trends, Event Type, Event Trends, Top Sender, Top Recipient, Top Subject, Client OS, Client Device Type, Client Type, Client Name, Event Geolocation, and Recent Event Summary. This dashboard empowers Mailgun users to analyze and optimize email delivery by monitoring event trends, identifying errors, understanding client behavior, and tracking event geolocation data, ensuring efficient email management and enhanced user experience.
+
+<img src={useBaseUrl('img/integrations/webhooks/mailgun_overview.png')} style={{border: '1px solid black'}} alt="Mailgun - Overview"/>
+
+### Mailgun - Failed, Complained Events
```
Comment (JV0812) [inline docs/integrations/webhooks/mailgun.md:126]:
```suggestion
### Failed, Complained Events
```

File: docs/integrations/webhooks/neoload.md [hunk 5]
```diff
@@ -7,12 +7,50 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 <img src={useBaseUrl('img/integrations/webhooks/neoload-logo.png')} alt="Thumbnail icon" width="50"/>
 
+
+The Sumo Logic app for Neoload provides a high-level perspective, enabling monitoring of test execution, completion, termination, and tests in progress. It also tracks targeted projects, tests, scenarios, and workspaces while assessing overall test and quality status. Also the dashboards delves deeper into performance testing, offering insights into test frequency, test organization by workspace and projects, various trends and tests with execution time and max virtual users (VUs). The app empower users to make data-driven decisions, optimize testing strategies, and enhance overall performance testing efficiency.
```
Comment (jpipkin1) [inline docs/integrations/webhooks/neoload.md:11]:
```suggestion
The Sumo Logic app for Neoload provides a high-level perspective, enabling monitoring of test execution, completion, termination, and tests in progress. It also tracks targeted projects, tests, scenarios, and workspaces while assessing overall test and quality status. Also the dashboards delve deeper into performance testing, offering insights into test frequency, test organization by workspace and projects, and various trends and tests with execution time and max virtual users (VUs). The app empowers users to make data-driven decisions, optimize testing strategies, and enhance overall performance testing efficiency.
```

File: docs/integrations/webhooks/netlify.md [hunk 6]
```diff
@@ -7,11 +7,120 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
 
 ## Event types
 
-The Sumo Logic integration for Netlify ingests site deployment events into Sumo Logic through an outgoing webhook available in Netlify. For more information on supported events that are ingested through the Netlify webhook, see the [Netlify Documentation](https://docs.netlify.com/site-deploys/notifications/)
+The Sumo Logic App for Netlify ingests site deployment events into Sumo Logic through an outgoing webhook available in Netlify. For more information on supported events that are ingested through the Netlify webhook, see the [Netlify Documentation](https://docs.netlify.com/site-deploys/notifications/)
```
Comment (jpipkin1) [inline docs/integrations/webhooks/netlify.md:16]:
```suggestion
The Sumo Logic app for Netlify ingests site deployment events into Sumo Logic through an outgoing webhook available in Netlify. For more information on supported events that are ingested through the Netlify webhook, see the [Netlify Documentation](https://docs.netlify.com/site-deploys/notifications/)
```

File: docs/integrations/webhooks/netlify.md [hunk 7]
```diff
@@ -42,10 +151,36 @@ Follow the below steps to configure the Netlify webhook.
 7. Click **Save**.
 8. Verify Netlify events are getting ingested in Sumo Logic by executing the following query on Sumo Logic's Log Search panel.
 ```sql
-`_sourceCategory=webhook/netlify`
+`_sourcecategory=webhook/netlify`
```
Comment (jpipkin1) [inline docs/integrations/webhooks/netlify.md:154]:
```suggestion
`_sourceCategory=webhook/netlify`
```

**Review by jpipkin1 — APPROVED** [review]
Looks good, @JV0812. I found only minor things.

**Comment by jpipkin1** [comment]
@JV0812, when will the v2 app PRs get published (this PR, #3432, #3433, #3428, and #3437)?

**Pull request merged**